src/Modules/Backend/Domain/UserGroup/UserGroup.php

```diff
@@ -90,7 +90,8 @@ public function getId(): int
 
     public static function fromDataTransferObject(UserGroupDataTransferObject $userDataTransferObject): self
     {
-        $userGroup = $userDataTransferObject->hasEntity() ? $userDataTransferObject->getEntity() : new self($userDataTransferObject->name);
+        $userGroup = $userDataTransferObject->hasEntity() ?
```

Comment:
not a fan of the ? at the end of the line, should be at the start of the next line for readability

Reply:
fixed 4e1261a65b9c1f3a82a284d86ab28b012142cc16